src/WikiService.Models.Dto/Requests/Article/GetArticleRequest.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc;
+
+namespace LT.DigitalOffice.WikiService.Models.Dto.Requests.Article
+{
+  public record GetArticleRequest
+  {
+    [FromQuery(Name = "articleId")]
+    public Guid? ArticleId { get; set; }
```

Comment:
why is it nullable?

Reply:
To cope with the case when there is nothing after "=" in "/get?articleId="

Reply:
get endpoint must contain the id of the entity to be received
the request "/get?articleId=" is invalid

Reply:
move the variable to GetAsync arguments